### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by JacobMcAloney**
Proposed update to paper.md to include additional details of the applications/tools as requested by the paper reviewers and outlined by @sjswerdloff and @Matthew-Jennings at https://github.com/pymedphys/pymedphys/issues/1667.

If the changes are too wordy or add to much volume I think it would be reasonable to trim the description of tools that make use of data extraction/conversion. I.e. lines 110 to 117 could be cut.

### CODE REVIEW

**Comment by SimonBiggs** [comment]
Thanks @JacobMcAloney, by the way, here's a link to the rendered paper:

https://github.com/pymedphys/pymedphys/suites/7052584802/artifacts/277878183

**Comment by SimonBiggs** [comment]
Beautiful stuff @JacobMcAloney, that reads beautifully 🙂

**Comment by sjswerdloff** [comment]
> Beautiful stuff @JacobMcAloney, that reads beautifully 🙂 

I second the motion.
[Applause]

**Review by Matthew-Jennings — APPROVED** [review]
Agree, thank you @JacobMcAloney!

**Comment by SimonBiggs** [comment]
@Matthew-Jennings, might you be interested in doing a judicious copy into the general README file?
